lib/gen_stage.ex

```diff
@@ -1196,10 +1178,12 @@ defmodule GenStage do
 
   ## Options
 
-    * `:cancel` - `:permanent` (default) or `:temporary`. When `:permanent`,
-      the consumer exits when the producer cancels or exits. In case
-      of exits, the same reason is used to exit the consumer. In case of
-      cancellations, the reason is wrapped in a `:cancel` tuple.
+    * `:cancel` - `:permanent` (default), `:transient` or `:temporary`.
+      When permanent, the consumer exits when the producer cancels or exits.
```

Comment:
For ease of reading, I'd make "permanent" code as in `:permanent`, and same for others. I think it helps when reading the docs.